lib/outbound/mail.js

```diff
@@ -50,7 +50,9 @@ const messages = {
   accountResetDeleted: message('ODK Central account password reset', '<html>Hello!<p>A password reset has been requested for this email address, but the account has been deleted.</p><p>If this message is unexpected, simply ignore it. Otherwise, please double check the email address given for your account, and try contacting your Central administrator.</p></html>'),
 
   // Notifies a user that their password has been changed
-  accountPasswordChanged: message('ODK Central account password change', '<html>Hello!<p>We are emailing because you have an ODK Central data collection account, and somebody has just changed its password.</p><p>If this was you, please feel free to ignore this email.</p><p>Otherwise, please contact your local ODK system administrator immediately.</p></html>')
+  accountPasswordChanged: message('ODK Central account password change', '<html>Hello!<p>We are emailing because you have an ODK Central data collection account, and somebody has just changed its password.</p><p>If this was you, please feel free to ignore this email.</p><p>Otherwise, please contact your local ODK system administrator immediately.</p></html>'),
+
+  backupFailed: message('ODK Central backup failed', '<html>Hello:<p>This is an automated system message to the listed ODK Central system administrator. ODK Central just attempted to perform a backup, but was unable to. Please visit the administration website and navigate to the Backups settings page for more information.</p></html>')
```

Comment:
> navigate to the Backups settings page

Is this not a link because we don't have access to the domain? Because putting routes here is potentially a maintenance issue? Something else?

Reply:
bc i'm a dummy

Reply:
actually i know this sounds silly but actually it's sort of hard to do bc to form the link i need to know the full domain name, and i don't really have access to that information there. how critical do you think this is?

Reply:
so that we can merge this code, since other prs depend on it, i'm just merging this now, and if we want to add this we can do it in a separate pr.

Reply:
I think it would be convenient but is not critical. Is it possible to use one of the other emails with a link as a starting point? Could it be a contributor-friendly issue that just exists for now?

Reply:
well it was the plumbing that was the problem so it's not really a copy paste problem. but actually apparently that value does exist there because magic somehow slash maybe i did that a way i wasn't supposed to, but either way it's already in there so i'll throw this in w block 11 misc final pr.